Modules/Microsoft365DSC/Modules/M365DSCReport.psm1

```diff
@@ -570,7 +578,7 @@ function Compare-M365DSCConfigurations
                 # object. By scanning against the destination we will catch properties that are not null on the source but not null in destination;
                 foreach ($propertyName in $destinationResource.Keys)
                 {
-                    if ($propertyName -notin @("ResourceName", "ResourceId", "Credential", "CertificatePath", "CertificatePassword", "TenantId", "ApplicationId", "CertificateThumbprint", "ApplicationSecret"))
+                    if ($propertyName -notin @('ResourceName', 'ResourceId', 'Credential', 'CertificatePath', 'CertificatePassword', 'TenantId', 'ApplicationId', 'CertificateThumbprint', 'ApplicationSecret'))
```

Comment:
Should we add 'ManagedIdentity' to this list as well?

Could you create a separate Array of these values so that we do not define them twice within the code.

Reply:
Done